src/assets/stylesheets/_social_cards.scss

```diff
@@ -40,11 +40,9 @@
   margin-bottom: 5px;
   padding: 0 8px;
   min-height: 30px;
+  line-height: 30px;
 }
-[data-scrivito-display-mode="editing"]
-  [data-scrivito-editors-placeholder]:empty:not(:focus)::before {
-  line-height: 18px !important;
-}
+.seo_card_input .input:focus { box-shadow: 0 0 0 2px #577fb6 inset; }
```

Comment:
* [ ] We have two borders around focused inputs, which looks weird and is different from other UI elements:

|  ![image](https://user-images.githubusercontent.com/986170/140744053-d1a4fe8f-353d-4017-a10c-d10e19a2b102.png)  | ![image](https://user-images.githubusercontent.com/986170/140743988-3820704b-e6e5-4adb-9133-fe417a2189ac.png)  |
| -- | -- |
| ![image](https://user-images.githubusercontent.com/986170/140743841-31523320-71c2-4a0d-ab1e-5b9fe358714d.png)  |  ![image](https://user-images.githubusercontent.com/986170/140743911-69aa03a3-9a03-489d-9e2c-96233d72382c.png)  |

* [ ] FYI Not caused by the current changes, but I noticed it here: The text input color for an input with a warning is not grey, as it is for regular UI inputs with warnings.